velox/exec/HashAggregation.cpp

```diff
@@ -135,9 +139,13 @@ void HashAggregation::addInput(RowVectorPtr input) {
   // NOTE: we should not trigger partial output flush in case of global
   // aggregation as the final aggregator will handle it the same way as the
   // partial aggregator. Hence, we have to use more memory anyway.
+  //
+  // We currently disable flushing when spilling is enabled. It's possible
```

Comment:
Perhaps, update comments for the new config to explain that it disables flushing.

Reply:
done